Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -769,23 +769,23 @@
 #define ENABLE_VORBIS 1
 #endif
 
-#if !defined(ENABLE_SET_WEBCONTENT_PROCESS_INFORMATION_IN_NETWORK_PROCESS) && PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000
+#if !defined(ENABLE_SET_WEBCONTENT_PROCESS_INFORMATION_IN_NETWORK_PROCESS) && PLATFORM(MAC)
 #define ENABLE_SET_WEBCONTENT_PROCESS_INFORMATION_IN_NETWORK_PROCESS 1
 #endif
 
 #if !defined(ENABLE_WEBM_FORMAT_READER) && PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 110300
```

Comment:
This SDK version check is also unneeded now.

Reply:
Yep, that's done in https://github.com/WebKit/WebKit/pull/15189